12_col_widths.qmd

```diff
@@ -0,0 +1,81 @@
+---
+title: "Setting Column Widths"
+format: html
+editor: visual
+---
+
+## Adjusting column Widths output
+
+
+```{r eval=FALSE}
+
+library(gtsummary)
+tbl <- gtsummary::tbl_summary(gtsummary::trial, by = trt, include = c(age, grade), missing = "always", label = list(age = "Age  \n\U00A0\U00A0linebreak in a cell")) |> add_overall()
+gt_table <- tbl$table_body |> 
+  gt::gt(caption = gt::md("This is the caption  \n[Used for cross referencing in Quarto]")) |> 
+  gt::fmt_markdown(columns = label) |> # need this to recognize the line break in the table body
+  # Column headers with line breaks
+  gt::cols_label(
+    stat_0 = gt::md("**Overall**"),
+    stat_1 = gt::md("**Drug A**  \nManual Line Break"), 
+    stat_2 = gt::md("**Drug B**  \nManual Line Break")
+  ) |> 
+  gt::tab_header(gt::md("This is the Title  \nwith a linebreak"), subtitle = "This is the Subtitle") |>
+  gt::cols_width(everything() ~ px(200)) #apply a 200 px width to all columns
+```
+
+### gt_save PDF
+
+```{r}
+gt::gtsave(gt_table, filename = "outputs/pdf/col_widths_gt_pdf.pdf")
+```
+
+[Preview the Output](outputs/pdf/col_widths_gt_pdf.pdf)
+- column width applied but the table is cut off if it doesn't fit in the page
+
+### rmarkdown rendered PDF
+
+```{r eval = FALSE}
+gt_table
+```
+
+[Preview the Output](outputs/pdf/col_widths_rmd_pdf.pdf)
+- column width applied but the table is cut off if it doesn't fit in the page
+
+### quarto rendered PDF
+
+```{r eval = FALSE}
+gt_table
+```
+
+[Preview the Output](outputs/pdf/col_widths_qmd_pdf.pdf)
+- column width applied but the table is cut off if it doesn't fit in the page
```

Comment:
can you update the widths using `gt::pct()` so they don't flow over the page? We just want to know if the Word and PDF outputs respect the specified widths.

Reply:
done! The results in the exported docs seem to be the same as the ones with the `px()` function.